### ISSUE TITLE
Error logging crash

### ISSUE THREAD

**Issue by kujenga** [diffeo/go-coordinate#13]
The following crash was seen in a live system. It appears that a nil logging object may have been passed into the method?

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x877352]

goroutine 29 [running]:
github.com/diffeo/go-coordinate/vendor/github.com/sirupsen/logrus.(*Logger).Error(0x0, 0xc420ec7eb0, 0x1, 0x1)
	/home/ubuntu/.go_workspace/src/github.com/diffeo/go-coordinate/vendor/github.com/sirupsen/logrus/logger.go:216 +0x22
main.Observe(0xe122e0, 0xc420018290, 0xe10d60, 0xc42031f2c0, 0x1bf08eb000, 0x0)
	/home/ubuntu/.go_workspace/src/github.com/diffeo/go-coordinate/cmd/coordinated/metrics.go:60 +0x1da
created by main.main
	/home/ubuntu/.go_workspace/src/github.com/diffeo/go-coordinate/cmd/coordinated/main.go:91 +0x7c9
```

**Comment by henrywallace** [comment]
Shouldn't it be impossible for that error to be `nil`? https://github.com/diffeo/go-coordinate/blob/master/cmd/coordinated/metrics.go#L59-L62

One way around this might be to `log.Errorf("%+v", err)`.

**Comment by kujenga** [comment]
Closed out by #14
